### ISSUE TITLE
Losing bought price and hotconfig doest allow update

### ISSUE THREAD

**Issue by Masterofcoinak** [taniman/profit-trailer#2033]
## ProfitTrailer info
Version: 2.14

Exchange:
- [ ] Bittrex
- [x] Binance
- [ ] Poloniex

Market:
- [ ] BTC
- [x] ETH
- [ ] USDT
- [ ] BNB
- [ ] XMR

Operating System:
- [x] Windows (please specify version)
- [ ] Ubuntu (please specify version)
- [ ] Other linux distro (please specify)
- [ ] Mac OS

## Expected Behavior
I want PT to take the average price I give it to

## Current Behavior
I have done several actions with my coins on Binance
- Splitting of Coins
- Sell of Coins and switching to other coins (setting average price accordingly)
- Using tools as PTDefender (i know its not supported)
Its getting pretty hard for PT to recalculate the price I want, but I am also calculating all
prices by myself and I want to set them into the software.
Unfortunately PT is overwriting the price in the newest Version.
I can even see that PT is taking my price, but after a while it changes it back.

## Troubleshooting
Setting the price is for me since a long time a problem. 
(also before PTDefender, to avoid this as reason for closing this topic)
I changed even the VPS. I tried several PT versions. I used a clean PT, but it doesnt work.
In the previous version it work partly, but in the 2.1.4 it doesnt work anymore.

![image](https://user-images.githubusercontent.com/36849981/44161749-c60aec80-a0f0-11e8-8cac-fdb42c64d57a.png)

after 30 seconds
![image](https://user-images.githubusercontent.com/36849981/44161789-e6d34200-a0f0-11e8-9c87-f4774e2e3830.png)

PT doenst show any problem.
![image](https://user-images.githubusercontent.com/36849981/44161820-fe122f80-a0f0-11e8-8258-6052723dd40f.png)

**Comment by taniman** [comment]
the bot will tell you. Please set
NEO_bought_price (number)
TRX_bought_price (number)
IOTA_bought_price (number)

Look in your unaccounted folder for those (numbers) and upload those here.

If you are doing lots of stuff with your coins on binance you should expect the prices to be incorrect and get lost. And you should wait untill all the bot caches are up to date from the changes you have done manually on the exchange before attempting to set a bought_price.

But I will wait for the files and then I can see if it's a bug or just something the bot cannot and will not cope with at the moment

**Comment by Masterofcoinak** [comment]
This 3 dont even show up in the unaccounted folder. Just BNB is shown 18 times. 
They are currently in the Dust, so maybe thats the reason. 
But I tried to set the price for BNB (not in dust), I would say 100 Times and it didnt even took the price for a sec.

[unaccounted.zip](https://github.com/taniman/profit-trailer/files/2291323/unaccounted.zip)

**Comment by taniman** [comment]
Well it's actually a know fact the BNB will lose it's price easily.  Because you are using it to pay for fees so the amounts don't match anymore.

But I see you have reserved some BNB. And this indeed is a known issue with reserved balance and setting a bought price. 

I have tried to fix both these in 2.1.5 that will be released hopefully tomorrow. But it surely no guarantee as long as BNB is being used for fee

**Comment by Masterofcoinak** [comment]
I can live with that for BNB. Would be nice if its solved, but not the most important thing for me. The other coins are much more important. Any idea, what could be the issue?

**Comment by Masterofcoinak** [comment]
Yes I am using VPS. To be honest my clock was not aligned, but I changed now all to UTC time.
I tried it again and same issue happens. The Coins in the dust take the price for short time and lose them again.

![image](https://user-images.githubusercontent.com/36849981/44198026-cc41ad00-a172-11e8-889e-dbfb073fa744.png)

**Comment by Masterofcoinak** [comment]
Coins currently in the Pairs and DCA keep the bought price, but not in the dust and since a lot of people start the use the delution strategy it is import to have the correct price in the dust too.
